sig-ci/README.md

```diff
@@ -0,0 +1,38 @@
+# SIG: CI
+
+This SIG is focused on CI (continuous integration) experience within Spinnaker.
```

Comment:
Is this intended to be a long-lived SIG, or more of a project-focused group with the goal of delivering the CI view, defining necessary contracts for providers, etc.?

Reply:
+1 to Andrew's question.

Reply:
@demobox To your point, this sig is intended to be a long-lived sig and not just project-focused. We plan to tackle all CI elements within Spinnaker - Igor, CI view/config, potentially adding a CI stage, and so on. 
We will definitely collaborate with the UI/UX sig when we'll be considering adding elements to the UI such as the CI view.

Reply:
Sounds exciting and thanks for the clarifications @gal-yardeni! Perhaps add that language in some way to the SIG description? 

+1 from me.

Reply:
Done. Thanks @aglover!

Reply:
Thanks for the additional details!